common/config/cli.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"bytes"
+	"os"
+	"strconv"
+
+	"gopkg.in/urfave/cli.v1"
+)
+
+const (
+	environmentDev  = "dev"
+	environmentProd = "prod"
```

Comment:
I didn't. That's for loading configuration from the environment in `api`, `router`, `indexer`, etc.